src/pages/contact.html

```diff
@@ -0,0 +1,81 @@
+                  <input type="email" name="email" class="mt-2 px-3 py-2 bg-white border shadow-sm border-slate-300 focus:outline-none focus:border-main focus:ring-main block w-full rounded-md sm:text-sm focus:ring-1"/>
+                  <p data-error="email" class="text-sm text-red-600 mt-2 hidden"></p>
+                </label>
+                <label class="mb-7 block">
+                  <span data-i18n="contact.phone_number" class="after:content-['*'] after:ml-0.5 after:text-red-500 block text-sm font-medium text-slate-700"></span>
+                  <input type="text" name="phoneNumber" class="mt-2 px-3 py-2 bg-white border shadow-sm border-slate-300 focus:outline-none focus:border-main focus:ring-main block w-full rounded-md sm:text-sm focus:ring-1"/>
+                  <p data-error="phoneNumber" class="text-sm text-red-600 mt-2 hidden"></p>
+                </label>
+                <label class="mb-7 block">
+                  <span data-i18n="contact.comments" class="after:content-['*'] after:ml-0.5 after:text-red-500 block text-sm font-medium text-slate-700"></span>
+                  <textarea name="comments" class="h-48 mt-2 px-3 py-2 bg-white border shadow-sm border-slate-300 focus:outline-none focus:border-main focus:ring-main block w-full rounded-md sm:text-sm focus:ring-1"></textarea>
+                  <p data-error="comments" class="text-sm text-red-600 mt-2 hidden"></p>
+                </label>
+                <button type="submit" class="text-white bg-main px-[30px] py-[10px] rounded-[20px] text-[14px] uppercase border border-main border-1 hover:bg-white hover:text-main"><span data-i18n="contact.send_contact"></span></button>
+              </form>
+            </div>
+            <div class="md:basis-1/2 md:p-[30px] p-0 basis-full mb-[20px]">
+              <img src="/assets/images/logo.webp">
+              <p class="text-[13px] py-[20px]"><span data-i18n="contact.dkt_intro"></span></p>
+              <div class="flex items-center gap-2 mb-[10px]">
+                <div class="text-[15px] text-main">
+                    <i class="fas fa-mobile-alt"></i>
+                </div>
+                <span data-i18n="footer.phone"></span>: (84-4) 66.558.868
+              </div>
+              <div class="flex items-center gap-2">
+                  <div class="text-[12px] text-main">
+                      <i class="fas fa-envelope"></i>
+                  </div>
+                  <span>Email: </span> info@dkt.com.vn
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+  </main>
+
+  <footer id="footer"></footer>
+
+  <script src="/assets/js/header.js"></script>
+  <script src="/assets/js/footer.js"></script>
+  <script src="/assets/js/formContact.js"></script>
+  <script src="/assets/js/i18n.js"></script>
+  <script src="/assets/js/breadcrumb.js"></script>
```

Comment:
```suggestion
  <script src="/assets/js/formContact.js"></script>
  <script src="/assets/js/breadcrumb.js"></script>
  <script src="/assets/js/i18n.js"></script>
```
I18n sẽ nằm ở cuối nha